assets/parquet/main.py

```
@@ -0,0 +1,154 @@
+ASSETS_DIR: The directory this file resides in, inside the container
+WORK_DIR: The in-container path to an accessible location e.g. "/home"
+Benchmark: Base class for benchmarks, has docker_execute to execute command within container
+logger: A logging.Logger
+"""
+
+PARQUET_DATA_PATH = "/home/hive-data"
+PARQUET_SCHEMA_NAME = "bench_schema"
+PARQUET_TABLE_NAME = "bench_table"
+class parquet_bench(Benchmark):
+    # add any parameters to the tool here
+    def __init__(self, dataset, mode='json string'):
```

Ruff (0.12.2)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)